recipients_app/lib/view/pages/main_app_page.dart

```diff
@@ -26,10 +26,13 @@ class _MainAppPageState extends State<MainAppPage> {
             // TODO display error message
             if (recipient == null) return;
 
+            final organization = context.read<AuthCubit>().state.organization!;
```

Comment:
Same here `organization!` can lead to NullPointerExceptions probably. Should we make it without `!` as `AccountPage` accepts `Organization?` ?

Reply:
fixed